internal/clientconn/conn.go

```diff
@@ -331,7 +332,12 @@ func (c *conn) run(ctx context.Context) (err error) {
 			}
 
 			if c.l.Enabled(ctx, diffLogLevel) {
-				c.l.Log(ctx, diffLogLevel, "Header diff:\n"+diffHeader+"\nBody diff:\n"+diffBody)
+				if len(diffBody) > 0 {
+					diffBody = strings.TrimSpace(diffBody)
```

Comment:
Does diffBody finish with `\n`? Trying to understand why this is necessary.

Reply:
Yeah, it prints trailing `\n`, and there's no way to disable that.